dev/breeze/doc/01_installation.rst

```diff
@@ -455,6 +455,15 @@ Breeze on POSIX-compliant systems (Linux, MacOS) can be automatically installed
 ``breeze`` with it and setting up autocomplete.
 
 
+Troubleshooting
```

Comment:
NIT: We have 04_troubleshooting.rst - maybe we should add it there and point out to troubleshooting doc from here